backend/backend/settings/base.py

```diff
@@ -143,6 +143,14 @@ def get_required_setting(
 
 DEFAULT_AUTH_USERNAME = os.environ.get("DEFAULT_AUTH_USERNAME", "unstract")
 DEFAULT_AUTH_PASSWORD = os.environ.get("DEFAULT_AUTH_PASSWORD", "unstract")
+SYSTEM_ADMIN_USER = os.environ.get("SYSTEM_ADMIN_USER", "unstract_admin")
+SYSTEM_ADMIN_PASSWORD = os.environ.get(
+    "SYSTEM_ADMIN_PASSWORD", "unstract_admin"
```

Comment:
@muhammad-ali-e please remove the default value and use `get_required_setting` function.

Reply:
updated with required changes

Reply:
updated with required changes